### PR TITLE
feat: flesh out linking error scenarios

### PR DESCRIPTION
**Pull request by lewisdaly**
No sequences yet, I just want to agree on the error conditions first, then I'll make some sequences.

### CODE REVIEW

File: docs/linking/README.md [hunk 1]
```diff
@@ -369,12 +369,47 @@ send back any sort of secret and no need to pass a secret back to the DFSP.
...
 # 4. Linking Error Scenarios
 
+
+Broadly speaking, the following error scenarios 
+
+1. `GET /accounts/{ID}`
+      1. DFSP cannot find any accounts linked for the username provided, or some other issue looking up accounts.
+1. `POST /consentRequests`
+      1. The list of scopes provided by the PISP in `POST /consentRequests` is unsuitable to the DFSP
+      1. The pisp provided a bad `callbackUri` that the DFSP doesn't trust
```
Comment (kleyow) [inline docs/linking/README.md:385]:
```suggestion
      1.1 The pisp provided a bad `callbackUri` that the DFSP doesn't trust
      1.2D FSP does not support any requested authentication channels
```

File: docs/linking/README.md [hunk 2]
```diff
@@ -369,12 +369,47 @@ send back any sort of secret and no need to pass a secret back to the DFSP.
...
+<!-- ## 3.2. Credential registration
 
-TODO!
+TODO! -->
 
 # 4. Linking Error Scenarios
 
+
+Broadly speaking, the following error scenarios 
+
```
Comment (kleyow) [inline docs/linking/README.md:380]:
```suggestion
1. GET /services/THIRD_PARTY_DFSP
    1.1 No thirdparty enabled DFSP's found
```

**Comment by kleyow** [comment]
All the other errors look good for a first pass 👍

**Review by eoln — APPROVED** [review]

**Comment by lewisdaly** [comment]
This looks wildly outdated

**Pull request closed without merging**